src/global-vars.scss

```diff
@@ -50,3 +50,48 @@ $header-search-bar-height: 70px;
 $footer-height: 120px;
 $minimum-body-margin: 20px;
 $media-breakpoint: 600px;
+
+// Mobile
+$xSmallMaxScreenSize: 599px;
+
+// Tablet
+$smallMaxScreenSize: 1239px;
+
+// Laptop
+$mediumMaxScreenSize: 1599px;
+
+/**
+ * Media Queries
+ *
+ * Example usage:
+ * ```
+ *     .my-div {
+ *       width: 100%; // Desktop
+ *     }
+ *
+ *     @include laptopMaxMediaQuery {
+ *        width: 200px; // Laptop
+ *     }
+ *
+ *     @include mobileMaxMediaQuery {
+ *        width: 100px; // Mobile
+ *     }
+ * ```
+ */
+@mixin laptopMaxMediaQuery {
+    @media only screen and (max-width: $mediumMaxScreenSize) {
+      @content;
+    }
+}
+  
+@mixin tabletMaxMediaQuery {
+    @media only screen and (max-width: $smallMaxScreenSize) {
+        @content;
+    }
+}
+
+@mixin mobileMaxMediaQuery {
```

Comment:
looks like this is the only media query that's being used, so you can delete the rest.
also we already have "$media-breakpoint: 600px" defined for mobile, so you could just use that variable here.

Reply:
done